app/repos/exemption.js

```diff
@@ -114,8 +114,8 @@ const handleOrder2023 = (registration, data) => {
   }
 }
 
-const handleOrder2015 = async (registration, data, cdo) => {
-  if (registration.exemption_order.exemption_order === '2015' && cdo?.status?.status !== constants.statuses.InterimExempt) {
+const handleCourt = async (registration, data, cdo) => {
+  if (data.court && data.court !== '') {
```

Comment:
Is there a test to demonstrate this?

Reply:
Good question. I'd assumed it was already covered but I'll check